designsafe/apps/api/messages/models.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import unicode_literals
+import logging
+from django.db import models
+
+logger = logging.getLogger(__name__)
+TYPE_CHOICES = (
+    ('alert-warning', 'WARNING'),
+    ('alert-danger', 'DANGER'),
+    ('alert-info', 'INFO'),
+    ('alert-success', 'SUCCESS'),
+    )
+
+class PageAlert(models.Model):
+    alert_id = models.CharField(max_length=120, unique=True, primary_key=True)
+    alert_type = models.CharField(max_length=20, choices=TYPE_CHOICES, default='alert-warning')
```

Comment:
Do we need to prepend each field with `alert_` the model class is already named `PageAlert` we could name them like:
```python
class PageAler(models.Model):
    type = models.CharField(max_length=20)
    message = models.TextField(help_text='Page alert message text.')
```

Reply:
I'll fix that